triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunner.scala

```diff
@@ -75,6 +78,12 @@ object TriggerRunner {
                 Behaviors.same
               case Stop =>
                 Behaviors.stopped // Automatically stops the child actor if running.
+              case UpdateToken(accessToken, refreshToken) =>
```

Comment:
It looks like currently, the actual trigger is stopped (without an error), the supervisor will keep running and then when we send the supervisor receives the message it starts the child again. Analogous to my comment elsewhere, why don’t we let the supervisor die and restart the whole thing from the server?

Reply:
See https://github.com/digital-asset/daml/pull/8037#discussion_r534048335